src/components/EventList.js

```diff
@@ -1,7 +1,9 @@
 import { FlashList } from '@shopify/flash-list';
 import PropTypes from 'prop-types';
 import React, { useEffect, useState } from 'react';
+import { StyleSheet } from 'react-native';
 
+import { normalize } from '../config';
 import { useRenderItem } from '../hooks';
```

Comment:
When I want to open the Ort dropdown filter in the event list, I get the following error.

![Simulator Screenshot - iPhone 15 Pro Max - 2024-07-29 at 09 02 09](https://github.com/user-attachments/assets/6bea1d9d-128e-4753-8644-8c1e56884891)

Reply:
Works for me, please try again with the latest code:

![Simulator Screenshot - iPhone 15 Plus - 2024-09-19 at 15 28 53](https://github.com/user-attachments/assets/f2f25a49-9023-4429-8d6a-98e109fccbc3)

Reply:
i found out, that the problem is connected to the screen height and containing list. somehow the dropdown cannot calculate the top position correctly to place the options directly under the box:

https://github.com/user-attachments/assets/23ce8d4b-89bc-4ed7-b20d-28764a3a0e2a

Reply:
fixed

https://github.com/user-attachments/assets/4f4df988-83ea-4a2d-bbdb-38ffacb58b82